plugins/mcreator-localization/lang/texts.properties

```diff
@@ -867,6 +867,8 @@ elementgui.common.page_generation=Generation
 elementgui.common.name_in_gui=Name in GUI:
 elementgui.common.rarity=Rarity:
 elementgui.common.stack_size=Stack size:
+elementgui.common.max_stack_size=Max stack size:
+elementgui.common.inventory_stack_size=Max size of stack:

```

Comment:
Please delete this variant of "Max stack size:" and use the shorter one instead.